android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -423,7 +423,7 @@ internal class CommonKtTests {
             mockPurchases.purchase(any<PurchaseParams>(), capture(capturedPurchaseCallback))
         } answers {
             val params = it.invocation.args.first() as PurchaseParams
-            assertEquals(false, params.isPersonalizedPrice)
+            assertEquals(null, params.isPersonalizedPrice)
```

Comment:
Should this be `assertNull` now?
But I'm confused why this boolean is nullable now?

Reply:
Will change to `assertNull` 💪 

And it was always nullable (see https://github.com/RevenueCat/purchases-android/blob/main/purchases/src/main/kotlin/com/revenuecat/purchases/PurchaseParams.kt#L11) but there was a bug where it was being default to `false` (so it was never `null`)

This was fixed between `6.1.1` and `6.4.0` which is why these tests are changing now 😊